apps/DemoApp/DemoApp/ViewController.m

```diff
@@ -283,7 +283,7 @@ - (void)setupAVPlayerViewController:(AVPlayer *)player {
                                                                                      viewData:viewData
                                                                                    customData:customData
                                                                                    viewerData:viewerData];
-    _playerBinding = [MUXSDKStats monitorAVPlayerViewController:_avplayerController withPlayerName:DEMO_PLAYER_NAME customerData:customerData];
+    _playerBinding = [MUXSDKStats monitorAVPlayerViewController:_avplayerController withPlayerName:DEMO_PLAYER_NAME customerData:customerData automaticErrorTracking:YES beaconCollectionDomain:@"myDomain.com"];
```

Comment:
we should remove this bc it isn't expected to work right?

Reply:
yep, my bad. Updated